docs/languages/typescript.md

```diff
@@ -107,7 +107,7 @@ You should now see the transpiled `HelloWorld.js` JavaScript file which you can
 
 ### Step 2: Run the TypeScript build
 
-Execute **Run Build Task...** from the global **Tasks** menu. If you created a `tsconfig.json` file in the earlier section, this should present the following picker:
+Execute **Run Build Task...** (`kb(workbench.action.tasks.build)`) from the global **Tasks** menu. If you created a `tsconfig.json` file in the earlier section, this should present the following picker:
```

Comment:
Can you also update **Tasks** menu to **Terminal** menu? The menu was renamed recently. Thanks

Reply:
I'll go a head and fix the **Tasks** menu issue. There are other places in the docs as well.